src/test/java/net/atos/entng/calendar/helper/CalendarHelperTest.java

```diff
@@ -77,33 +77,16 @@ public void testGetICalFromExternalPlatform_normalUse(TestContext context) throw
         String host = "";
         String i18nLang = "";
 
-
-        // Expected data
-        JsonObject expectedEvent = new JsonObject()
-                .put("owner.userId", USER_ID)
-                .put("is_default", true);
-
         Mockito.doAnswer(invocation -> {
             String address = invocation.getArgument(0);
             context.assertEquals(address, "fr.openent.zimbra");
-            Handler<AsyncResult<Message<JsonObject>>> replyHandler = invocation.getArgument(2);
-            replyHandler.handle(Future.succeededFuture(new ResultMessage(new JsonObject().put("result", new JsonObject().put("ics", "icsResult")))));
+            async.complete();
 
             return null;
         }).when(eventBus).request(Mockito.any(), Mockito.any(), Mockito.any());
 
-        Mockito.doAnswer(event -> {
```

Comment:
event bus only returns success result or fail, no ical strign so there is no ical to process